--- libsamplerate-0.1.8/src/src_sinc.c.orig	2013-08-11 18:07:35.000000000 -0500
+++ libsamplerate-0.1.8/src/src_sinc.c	2013-08-11 18:07:59.000000000 -0500
@@ -49,7 +49,6 @@
 
 #include "fastest_coeffs.h"
 #include "mid_qual_coeffs.h"
-#include "high_qual_coeffs.h"
 
 typedef struct
 {	int		sinc_magic_marker ;
@@ -116,9 +115,7 @@
 sinc_get_name (int src_enum)
 {
 	switch (src_enum)
-	{	case SRC_SINC_BEST_QUALITY :
-			return "Best Sinc Interpolator" ;
-
+	{	
 		case SRC_SINC_MEDIUM_QUALITY :
 			return "Medium Sinc Interpolator" ;
 
@@ -141,9 +138,6 @@
 		case SRC_SINC_MEDIUM_QUALITY :
 			return "Band limited sinc interpolation, medium quality, 121dB SNR, 90% BW." ;
 
-		case SRC_SINC_BEST_QUALITY :
-			return "Band limited sinc interpolation, best quality, 145dB SNR, 96% BW." ;
-
 		default :
 			break ;
 		} ;
@@ -211,12 +205,6 @@
 				temp_filter.index_inc = slow_mid_qual_coeffs.increment ;
 				break ;
 
-		case SRC_SINC_BEST_QUALITY :
-				temp_filter.coeffs = slow_high_qual_coeffs.coeffs ;
-				temp_filter.coeff_half_len = ARRAY_LEN (slow_high_qual_coeffs.coeffs) - 1 ;
-				temp_filter.index_inc = slow_high_qual_coeffs.increment ;
-				break ;
-
 		default :
 				return SRC_ERR_BAD_CONVERTER ;
 		} ;
